### PR TITLE
Airlock

### PR DESCRIPTION
**Pull request by lizhennessy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,27 +6,26 @@ Welcome to the starter code for **Federation with TypeScript**. You can find the
...
 
-In order to install and run the project locally, run:
+In order to install and run the project locally, navigate into the `listings` directory and run:
 
 ```shell
 npm install && npm run dev
 ```
 
-Right now, the project is a GraphQL server returning playlist and track data at `http://localhost:4000`. You can use visit `http://localhost:4000` directly, or use [Apollo Sandbox](https://studio.apollographql.com/sandbox?endpoint=http://localhost:5059/graphql) to connect to the endpoint and send queries.
+Right now, `listings` is a GraphQL server returning listing and amenity data at `http://localhost:4000`. You can use visit `http://localhost:4000` directly, or use [Apollo Sandbox](https://studio.apollographql.com/sandbox?endpoint=http://localhost:5059/graphql) to connect to the endpoint and send queries.
```
Comment (mabuyo) [inline README.md]:
```suggestion
Right now, `listings` is a GraphQL server returning listing and amenity data at `http://localhost:4000`. You can visit `http://localhost:4000` directly, or use [Apollo Sandbox](https://studio.apollographql.com/sandbox?endpoint=http://localhost:5059/graphql) to connect to the endpoint and send queries.
```

File: listings/src/datasources/listing-api.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    return this.get<Listing>(`listings/${listingId}`);
+  }
+
+  getAmenities(listingId: string): Promise<Amenity[]> {
+    console.log("Making a follow-up call for amenities with ", listingId);
+    return this.get<Amenity[]>(`listings/${listingId}/amenities`)
+  }
+  createListing(listing: CreateListingInput): Promise<Listing> {
+    return this.post("listings", {
+      body: { listing}
```
Comment (mabuyo) [inline listings/src/datasources/listing-api.ts]:
```suggestion
      body: { listing }
```
prettier file would catch this!

File: listings/src/schema.graphql [hunk 3]
```diff
@@ -0,0 +1,61 @@
+"A particular intergalactic location available for booking"
+type Listing {
+  id: ID!
+  "The listing's title"
+  title: String!
+  description: String!
```
Comment (mabuyo) [inline listings/src/schema.graphql:7]: This doesn't have a description?

File: listings/src/schema.graphql [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+  title: String!
+  "The listing's description"
+  description: String!
+  "The number of beds available"
+  numOfBeds: Int!
+  "The cost per night"
+  costPerNight: Float!
+  "Indicates whether listing is closed for bookings (on hiatus)"
+  closedForBookings: Boolean
+  "The Listing's amenities"
```
Comment (mabuyo) [inline listings/src/schema.graphql]:
```suggestion
  "The listing's amenities"
```

File: listings/src/schema.graphql (hunk 3 above)
Comment (lizhennessy) [inline listings/src/schema.graphql:7]:
```suggestion
"The listing's description"
  description: String!
```

File: listings/src/schema.graphql [hunk 5]
```diff
@@ -0,0 +1,62 @@
+"A particular intergalactic location available for booking"
+type Listing {
+  id: ID!
+  "The listing's title"
+  title: String!
+"The listing's description"
```
Comment (lizhennessy) [inline listings/src/schema.graphql]:
```suggestion
    "The listing's description"
```

**Review by mabuyo — APPROVED** [review]

**Pull request merged**